snuba/datasets/metrics_bucket_processor.py

```diff
@@ -101,10 +113,13 @@ def _should_process(self, message: Mapping[str, Any]) -> bool:
 
     def _process_values(self, message: Mapping[str, Any]) -> Mapping[str, Any]:
         values = message["value"]
-        for v in values:
-            assert isinstance(
-                v, (int, float)
-            ), "Illegal value in set. Int expected: {v}"
```

Comment:
Some of these comments mismatched what the check was, take this one for example. It's asserting and `int` or `float` but comment says "Int expected". I updated these wherever I found a discrepancy, should be reviewed.